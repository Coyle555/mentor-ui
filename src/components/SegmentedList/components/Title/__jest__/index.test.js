import React from 'react';
import { Title } from '../index';
import renderer from 'react-test-renderer';

test('Rendering title with just the title', () => {
	const tree = renderer.create(<Title title="test" />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Rendering title with just inserting functionality', () => {
	const tree = renderer.create(
		<Title insertable={true} onInsertClick={() => {}} />
	).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Rendering with both title and insertability', () => {
	const tree = renderer.create(
		<Title
			insertable={true}
			onInsertClick={() => {}}
			title="test"
		/>
	).toJSON();

	expect(tree).toMatchSnapshot();
});
