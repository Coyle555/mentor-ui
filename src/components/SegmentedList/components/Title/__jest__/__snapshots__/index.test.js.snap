// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering title with just inserting functionality 1`] = `
<div
  className="title-container"
>
  <button
    className="APMButton insert-btn"
    onClick={[Function]}
    type="button"
  >
    <i
      className="fas fa-plus"
    />
     Add Item
  </button>
</div>
`;

exports[`Rendering title with just the title 1`] = `
<div
  className="title-container"
>
  <h2
    className="title"
  >
    test
  </h2>
</div>
`;

exports[`Rendering with both title and insertability 1`] = `
<div
  className="title-container"
>
  <h2
    className="title"
  >
    test
  </h2>
  <button
    className="APMButton insert-btn"
    onClick={[Function]}
    type="button"
  >
    <i
      className="fas fa-plus"
    />
     Add Item
  </button>
</div>
`;
